scripts/database.js

```diff
@@ -358,10 +370,12 @@ export class Database {
                     shieldOutput: tx.shieldOutput,
                     bindingSig: tx.bindingSig,
                     lockTime: tx.lockTime,
+                    txid: tx.txid,
                 });
             })
             .sort((a, b) => a.blockHeight - b.blockHeight);
```

Comment:
A quick performance comment that is out-of-scope for this PR: ideally we should store transactions chronologically, so that we don't need to sort them at every load, this again is a fairly expensive operation that is avoidable if we store them properly ordered in the first place.

Even if we store the transactions in a chronologically-reversed format, we can scroll the cursor backwards with `openCursor(null, "prev")` thus having zero impact on performance, only gains.